lib/sentry.dart

```diff
@@ -494,3 +503,445 @@ class User {
     };
   }
 }
+
+/// The context interfaces provide additional context data.
+/// Typically this is data related to the current user,
+/// the current HTTP request.
+class Contexts {
+  /// This describes the device that caused the event.
+  final Device device;
+
+  /// Describes the operating system on which the event was created.
+  /// In web contexts, this is the operating system of the browse
+  /// (normally pulled from the User-Agent string).
+  final Os os;
```

Comment:
Let's spell is `OperatingSystem` and `operatingSystem`. It's longer but less ambiguous.